packages/ui-components/src/components/text-field/text-field.tsx

```diff
@@ -106,11 +111,13 @@ export class KvTextField implements ITextField, ITextFieldEvents {
 	}
 
 	componentDidLoad() {
-		if (this.type === EInputFieldType.Number) {
+		if (this.getInputMaskEnabled()) {
 			return Inputmask({
 				...NUMERIC_TEXT_INPUT_MASK_CONFIG,
```

Comment:
change to `TEXT_INPUT_MASK_CONFIG` this config can be used for any type not only numeric